### PR TITLE
Fix middlewares on update

### PR DESCRIPTION
**Pull request by paganotoni**
This PR is part of the movement of the middlewares separate packages, this will make the fix/upgrade command update those imports.

### CODE REVIEW

File: buffalo/cmd/fix/middleware.go [hunk 1]
```diff
@@ -0,0 +1,157 @@
...
+	if err != nil {
+		return err
+	}
+
+	newContents := string(read)
+	newContents = strings.Replace(newContents, "middleware.SetContentType", "contenttype.Set", -1)
+	newContents = strings.Replace(newContents, "middleware.AddContentType", "contenttype.Add", -1)
+	newContents = strings.Replace(newContents, "middleware.ParameterLogger", "paramlogger.ParameterLogger", -1)
+	newContents = strings.Replace(newContents, "middleware.PopTransaction", "popmw.Transaction", -1)
+
```
Comment (kteb) [inline buffalo/cmd/fix/middleware.go]: it's missing `newContents = string.Replace(newContents, "middleware.ForceSSL", "ssl.Middleware", -1)`

Comment (paganotoni) [reply buffalo/cmd/fix/middleware.go]: Oh, you are right, i only replaced the "root" middleware expressions but may need to also replace the ones using packages as well, i'll also add some tests.

Comment (paganotoni) [reply buffalo/cmd/fix/middleware.go]: Thanks for the review.

**Comment by kteb** [comment]
Thanks it looks great, I tried it in an app and it went well this time ;).

**Pull request merged**